### PR TITLE
makes sure ulimit can be set high

### PR DESCRIPTION
**Pull request by javierwilson**
for nginx, pgbouncer and redis

### CODE REVIEW

File: ansible/vars/dev/dev_public.yml [hunk 1]
```diff
@@ -127,6 +127,9 @@ backup_blobdb: True
 postgres_s3: False
 nadir_hour: 18
 
+nofile_limit: 65536
+nginx_worker_rlimit_nofile: "{{ nofile_limit }}"
```
Comment (snopoke) [inline ansible/vars/dev/dev_public.yml]: where are these used?

File: ansible/roles/postgresql/templates/pgbouncer.init.j2 [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+  status)
+    status_of_proc -p $PIDFILE $DAEMON $NAME && exit 0 || exit $?
+    ;;
+
+  *)
+    echo "Usage: /etc/init.d/$NAME {start|stop|reload|force-reload|restart|status}"
+    exit 1
+    ;;
+esac
```
Comment (snopoke) [inline ansible/roles/postgresql/templates/pgbouncer.init.j2]: What's different in this from the standard one?

File: ansible/vars/dev/dev_public.yml (hunk 1 above)
Comment (javierwilson) [reply ansible/vars/dev/dev_public.yml]: nofile_limit is the general max files open and nginx_worker_rlimit_nofile only for nginx max files open,  maybe this one is redudant?

Comment (snopoke) [reply ansible/vars/dev/dev_public.yml]: OK, I thought you had introduced these in this PR but I see they are already in use.

File: ansible/roles/postgresql/templates/pgbouncer.init.j2 (hunk 2 above)
Comment (javierwilson) [reply ansible/roles/postgresql/templates/pgbouncer.init.j2]: it applies ulimit if $ULIMIT exists (usually this means it has been set in /etc/defaults) I am copying this from how other services do it (redis, nginx)

**Pull request merged**